ApollosProject/apollos-apps
# feat: add date to content items

## DESCRIPTION
### What does this PR do, or why is it needed?
This PR adds `publishDate` to the content item schema so that we have the ability to show dates on content items in the future.

### How do I test this PR?
Start the server by running `TZ=UTC yarn start`

In graphql, query to find a content item id
![Screen Shot 2020-11-18 at 10 09 58 AM](https://user-images.githubusercontent.com/52924611/99548034-37980c00-2986-11eb-9c10-572b5d605cf5.png)

Use that content item id to query for `publishDate` and 
1. Confirm that the queried `publishDate` returns the same date as running `new Date ("publishDate")` in the console &
2. Confirm `publishDate` is a UTC string from the API (four hours different from the string returned in Rock, as time stamp is assumed Eastern).

<img width="1389" alt="Screen Shot 2020-11-24 at 10 41 54 AM" src="https://user-images.githubusercontent.com/52924611/100117798-2d24b900-2e43-11eb-8f4b-ecb8fee49280.png">

<img width="381" alt="Screen Shot 2020-11-24 at 10 50 47 AM" src="https://user-images.githubusercontent.com/52924611/100118378-c8b62980-2e43-11eb-8799-d82a20ec424a.png">

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
